ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -1011,15 +1047,15 @@ end
 `<%= render partial: 'user_info', object: { name: 'user' } %>`
 
 - [ ] `<%= locals.user_info.name %>`
-- [x] `<%= object.name %>`
+- [ ] `<%= object.name %>`
```

Comment:
Hash, in Ruby or Rails, does not accept dot notation, an OpenStruct does, but I checked the type of locals and it's just a Hash. I also tested this, and `object.name` returns an error.

Because of this, I decided to create a new (correct) option.